buildSrc/src/main/kotlin/publish-conventions.gradle.kts

```diff
@@ -0,0 +1,42 @@
+plugins {
+    `maven-publish`
+    signing
+}
+
+// empty xxx-javadoc.jar
+val javadocJar = tasks.register<Jar>("javadocJar") {
+    archiveClassifier.set("javadoc")
+}
+
+publishing {
+    repositories {
+        configureMavenPublication(this, project)
+    }
+
+    publications.withType<MavenPublication>().all {
+        pom.configureMavenCentralMetadata(project)
```

Comment:
Same comment as with `configureMavenPublication()` above -  this is th only usage of the `configureMavenCentralMetadata()` util. Could it be inlined?

Reply:
inlined